pkg/app/piped/executor/ecs/deploy.go

```diff
@@ -82,6 +82,23 @@ func (e *deployExecutor) ensureSync(ctx context.Context) model.StageStatus {
 	if !ok {
 		return model.StageStatus_STAGE_FAILURE
 	}
+
+	if e.appCfg.Input.IsStandaloneTask() {
```

Comment:
How about defining `escInput := e.appCfg.Input` at the beginning of this method?

Reply:
That sounds great🚀
It's easy to understand.